### PR TITLE
make pulling sweeps optional when using public api to query for runs

### PR DESCRIPTION
**Pull request by kptkin**
Fixes WB-9297

Description
-----------
What does the PR do?

This PR allows users to specify if they want to pull the sweeps (when pulling the runs) with the `Api.runs` method. The default stays `True` for backward compatibility, however the user could specify `Api.runs(..., include_sweeps=False)` in case they don't need sweeps and save time. From my simple example it seems to reduce the fetch time by ~3.6-5.9x. 

Testing
-------
How was this PR tested?

Manually

### CODE REVIEW

File: wandb/apis/public.py [hunk 1]
```diff
@@ -755,7 +755,14 @@ def users(self, username_or_email):
         res = self._client.execute(self.USERS_QUERY, {"query": username_or_email})
         return [User(self._client, edge["node"]) for edge in res["users"]["edges"]]
 
-    def runs(self, path=None, filters=None, order="-created_at", per_page=50):
+    def runs(
+        self,
+        path: Optional[str] = None,
+        filters: Optional[str] = None,
+        order: str = "-created_at",
```
Comment (dmitryduev) [inline wandb/apis/public.py:762]: Next time we touch this, it will become Literal :)

**Review by dmitryduev — APPROVED** [review]
LGTM!

**Comment by dmitryduev** [comment]
One comment: can we properly test this now?

**Comment by kptkin** [comment]
> One comment: can we properly test this now?

Do you mean performance test? Or that we don’t pull the sweep info?

**Comment by dmitryduev** [comment]
@kptkin: just that we don't pull in the info

**Pull request merged**